packages/host/app/components/ai-assistant/toast.gts

```diff
@@ -223,7 +234,9 @@ export default class AiAssistantToast extends Component<Signature> {
 
   @action
   private viewInChat() {
+    this.state.isResetStateValueBlocked = true;
```

Comment:
Adding this in viewChat to fix the flaky test that @lukemelia mentioned on Discord. If we don't block the reset state process, the `this.roomId` value can be empty when saving `currentRoomIdPersistenceKey`.